app/routes.py

```diff
@@ -1,2 +1,196 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify
+from datetime import datetime
+import requests
+from app import slack_key
+from app.models.goal import Goal
 
+task_list_bp = Blueprint("task_list", __name__, url_prefix = "/tasks")
+
+@task_list_bp.route("", methods = ["POST"])
+def create_task():
+    request_body = request.get_json()
+    attributes = ["title", "description", "completed_at"]
+
+    for attribute in attributes:
+        if attribute not in request_body:
+
+            return make_response({"details": "Invalid data"}), 400
+
+    new_task = Task(title = request_body["title"],
+                description = request_body["description"],
+                completed_at = request_body["completed_at"])
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return jsonify({"task": new_task.to_json()}), 201
```

Comment:
For readability I recommend using the same format to create your responses throughout your project.  The docs for `make_response` state that dictionaries passed to `make_response` will be converted to JSON.  Either format is fine, but having multiple formats can make it seem that different things are happening when in fact it's essentially the same.